bedrock/firefox/templates/firefox/new/email/scene2.html

```diff
@@ -0,0 +1,132 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-protocol.html" %}
```

Comment:
If I set `Dev=False`, I don't see the language switcher in the footer for this page, so i think we need to add `{% add_lang_files "firefox/new/quantum" %}` here?